packages/server/test/support/helpers/e2e.coffee

```diff
@@ -315,16 +315,18 @@ module.exports = {
         ## if we have browser in the stdout make
         ## sure its legit
         if matches = browserNameVersionRe.exec(stdout)
-          [str, key, browserName, version, headless] = matches
+          [str, key, customPath, browserName, version, headless] = matches
```

Comment:
now that i see what you're doing with this... there is likely an easier way to fix this. if this is the only single e2e test that needs this functionality then it might be better to tap into another seam here rather than account for it in all snapshots.

there's a callback method called `onBeforeStdout` (or something like that) in the `e2e.exec` function that will yield you the `stdout` prior to snapshotting that will enable you to modify stdout.

I would then use that callback function to replace the custom browser path you dynamically passed in so the snapshots are consistent.